### ISSUE TITLE
Migrations do not contain column definitions

### ISSUE THREAD

**Issue by carnedv** [beckenrode/mysql-workbench-export-laravel-5-migrations#41]
Using MySQL Workbench latest (`6.3.9`) on Windows 10 and the latest version of the script on `master` the generated migrations do not contain any column definitions.

With the MWB file linked here:
https://dl.dropboxusercontent.com/u/12420042/test.mwb
 (which is just a simple test with a single table with a PK and `varchar` field) the resulting migration is:

```
<?php

use Illuminate\Database\Schema\Blueprint;
use Illuminate\Database\Migrations\Migration;

class CreateTracksTable extends Migration
{
    /**
     * Run the migrations.
     * @table tracks
     *
     * @return void
     */
    public function up()
    {
        Schema::create('tracks', function (Blueprint $table) {
            $table->engine = 'InnoDB';
        });
    }

    /**
     * Reverse the migrations.
     *
     * @return void
     */
     public function down()
     {
       Schema::dropIfExists('tracks');
     }
}
```

I thought that maybe the MWB application version was too new so tried downgrading from `6.3.8` progressively to `6.3.5` but the migration generated was the same in each version. So either there's a bug or I'm missing something obvious on my end.

**Comment by carnedv** [comment]
Confirmed this same behavior on Mac with Sierra `10.12.3` using MWB `6.3.9`, so it's not something limited to a Windows environment.

**Comment by rmontanana** [comment]
Same here, hope this helps. Mac OS Sierra 10.12.3 & MySQL Workbench 6.3.8 Community 64 bits.
![issue](https://cloud.githubusercontent.com/assets/1431097/23865984/b7c729f4-0817-11e7-806e-e44ab8324d82.png)

**Comment by robvankeilegom** [comment]
Same behavior on Ubuntu 16.04 LTS 64-bit, MWB 6.3.9

I made a fix in: #43

**Comment by bandomesic** [comment]
I have a same problem Win 10 MWB 6.3.7

#43

**Comment by beckenrode** [comment]
Closing; Fixed in #43
